share/commands/agent-reset

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 # @description reset agent status and cache
+# @man Remove all locks and state cache of the agent, and restore initial promises.
+# @man This won't affect the desired state of the node, but will only
+# @man reset the internal state of the agent, for example to test a rule
+# @man without caching interference.
```

Comment:
You can add a little sentence about using it when there is a problem with promise update.